src/browser.rs

```diff
@@ -79,8 +80,11 @@ impl Browser {
     /// This fails if no web socket url could be detected from the child
     /// processes stderr for more than the configured `launch_timeout`
     /// (20 seconds by default).
-    pub async fn launch(config: BrowserConfig) -> Result<(Self, Handler)> {
-        // launch a new chromium instance
+    pub async fn launch(mut config: BrowserConfig) -> Result<(Self, Handler)> {
+        // Canonalize paths to reduce issues with sandboxing
+        config.executable = utils::canonicalize(&config.executable).await?;
+
```

Comment:
I guess we could also just use fs here, but now that we have an async version let's keep it